lobby/src/test/java/org/triplea/lobby/server/db/UserControllerIntegrationTest.java

```diff
@@ -37,52 +36,50 @@ void testCreate() throws Exception {
 
   @Test
   void testGet() {
-    final DBUser user = newUserWithMd5CryptHash();
-    assertEquals(user, controller.getUserByName(user.getName()));
+    final String user = newUserWithMd5CryptHash();
+    assertEquals(generateEmailAddress(user), controller.getUserEmailByName(user));
   }
 
   @Test
   void testDoesUserExist() {
-    assertTrue(controller.doesUserExist(newUserWithMd5CryptHash().getName()));
-    assertTrue(controller.doesUserExist(newUserWithBCryptHash().getName()));
+    assertTrue(controller.doesUserExist(newUserWithMd5CryptHash()));
+    assertTrue(controller.doesUserExist(newUserWithBCryptHash()));
   }
 
   @Test
   void testCreateDupe() {
-    final DBUser user = newUserWithMd5CryptHash();
+    final String user = newUserWithMd5CryptHash();
     assertThrows(
         Exception.class,
         () ->
             controller.createUser(
-                user.getName(),
-                user.getEmail(),
+                user,
+                generateEmailAddress(user),
                 new HashedPassword(md5Crypt(TestUserUtils.newUniqueTimestamp()))),
         "Should not be allowed to create a dupe user");
   }
 
   @Test
   void testLogin() {
     final String password = md5Crypt(TestUserUtils.newUniqueTimestamp());
-    final DBUser user = newUserWithHash(password, Function.identity());
+    final String user = newUserWithHash(password, Function.identity());
     controller.updateUser(
-        user.getName(), user.getEmail(), new HashedPassword(bcrypt(obfuscate(password))));
-    assertTrue(controller.login(user.getName(), new HashedPassword(password)));
-    assertTrue(controller.login(user.getName(), new HashedPassword(obfuscate(password))));
+        user, generateEmailAddress(user), new HashedPassword(bcrypt(obfuscate(password))));
+    assertTrue(controller.login(user, new HashedPassword(password)));
+    assertTrue(controller.login(user, new HashedPassword(obfuscate(password))));
   }
 
   @Test
   void testUpdate() throws Exception {
-    final DBUser user = newUserWithMd5CryptHash();
```

Comment:
I love this change 👍 

Took me a while to figure out that this refactoring dropped this class entirely

Reply:
I find it interesting that I added `DBUser` to begin with and have wanted to remove it now for some time. Just interesting to see my own style evolve over a couple of years and to see what I once thought was a good pattern for type safety to not be so good.

Reply:
Well I think the biggest (and perhaps only) problem is that really a DB-entity as you'd expect from a framework like spring. So you had to constantly sync this object with the actual state of the database which added a lot of boilerplate.

TL;DR; If the object itself would've modified the database based on its state and vice versa it would've probably been better